news/scripts/news_cluster.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import datetime
+
+import pandas as pd
+from sklearn.cluster import AgglomerativeClustering
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+from models.news import Cluster
+from mongo_news_db.db_connector import (delete_clusters, fetch_all,
+                                        insert_cluster)
+
+SECONDS_IN_DAY = 24 * 3600
+
+
+def main():
+    delete_clusters()
```

Comment:
This should be done after parsing the arguments. Right now calling `python news_cluster.py -h` will wipe your db

Reply:
good catch